lib/auth/providers.ts

```
@@ -10,21 +10,41 @@ export const providers = [
   GoogleProvider({
     clientId: process.env.GOOGLE_CLIENT_ID!,
```

Biome (1.9.4)

[error] Forbidden non-null assertion.



(lint/style/noNonNullAssertion)